src/Main.hs

```diff
@@ -176,6 +182,7 @@ options = OptSpec
     , timeout = Nothing
     , produceXml = False
     , printVersion = False
+    , zeroBasedCex = False
```

Comment:
I think it'll help with integration if either the default is `True` or if the `--xml` flag sets it to true.